InformationScripting/src/queries/VersionControlQuery.cpp

```diff
@@ -114,7 +114,27 @@ Optional<TupleSet> VersionControlQuery::executeLinear(TupleSet input)
 						changedNode = *ancestorIt;
 
 					if (outputNodesOnly)
+					{
 						result.add({{"ast", changedNode}});
+						switch (change->type())
+						{
+							case ChangeType::Deletion:
```

Comment:
Hmm, in general it's not that great if this query does too much. In particular we should probably make it just output the type of change as string, and leave coloring to another query.